### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Camilovelag**
## Good day! 💯 

This pull request is intended to add forms.

### 🛠️ Changes made

- [ ] Create a method called `current_user`.
- [ ] Create forms to perform the following functions: Create a `Post` and `Comment`.
- [ ] Allow Users to add likes to Posts.

I'm expecting feedback.

Happy coding!  👏👏👏

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @comment.post_id = params[:post_id]
+    @comment.author_id = current_user.id
+    respond_to do |format|
+      format.html do
+        if @comment.save
+          flash[:success] = 'Comment created succesfully'
+          redirect_to user_post_path(current_user, @comment.post_id)
+        else
+          flash[:error] = 'Error: Not able to create post'
+          render :new, locals: { comment: @comment }
```
Comment (Davidosky007) [inline app/controllers/comments_controller.rb:20]: ## OPTIONAL
* I strongly suggest you add a `comment_params ` method, This is a better security practice to help prevent accidentally allowing users to update sensitive model attributes. [Read more here about strong params](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -6,5 +6,36 @@ def index
...
+    @post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @post } }
+    end
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
+    @post.author_id = current_user.id
+    respond_to do |format|
```
Comment (Davidosky007) [inline app/controllers/posts_controller.rb:22]: ## OPTIONAL
* I strongly suggest you add a `post_params ` method, This is a better security practice to help prevent accidentally allowing users to update sensitive model attributes. [Read more here about strong params](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -6,5 +6,36 @@ def index
...
+        end
+      end
+    end
+  end
+
+  def like
+    @post = Post.find(params[:post_id])
+    @user = Post.find(params[:user_id])
+    Like.create(author_id: @user.id, post_id: @post.id)
+    redirect_to user_post_path(@user, @post)
```
Comment (Davidosky007) [inline app/controllers/posts_controller.rb]: * To implement this requirement `Allow Users to add likes to Posts` you need to create a `like controller` that allows the current user to like a post like you did for comments, Kindly fix this 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Camilovelag 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (Camilovelag) [reply app/controllers/posts_controller.rb:22]: I'm applying your suggestion tomorrow😄

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (Camilovelag) [reply app/controllers/comments_controller.rb:20]: I'm applying your suggestion tomorrow😄

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (Camilovelag) [reply app/controllers/posts_controller.rb]: Done! 😃

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Camilovelag ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**